lib/pub_sub/tasks/debug.rake

```diff
@@ -15,13 +15,13 @@ namespace :pub_sub do
       args.with_defaults(filter: nil, region: nil)
       puts 'Topics: ', '----------'
       PubSub.config.regions.each do |region|
-        next if !args[:region].blank? && region != args[:region]
+        next if args[:region].present? && region != args[:region]
         client = Aws::SNS::Client.new(region: region)
-        topics = collect_all(client, :topics)
-        topics.select!{|t| t.topic_arn =~ Regexp.new(args[:filter])} unless args[:filter].blank?
+        topics = PubSub::Subscriber.collect_all(client, :topics)
+        topics.select!{|t| t.topic_arn =~ Regexp.new(args[:filter])} if args[:filter].present?
```

Comment:
We should be able to do this check in the `#each` below and just `next` if criteria is not met.